third_party/proxy/README

```diff
@@ -0,0 +1 @@
+Package upstream is copied from [k8s.io/kubernetes@/v1.17.6](https://github.com/kubernetes/kubernetes/tree/v1.17.6) to avoid importing the whole kubernetes repo.
```

Comment:
what is package "upstream"? is that the old name before you moved the code to `third_party/`?

Reply:
Yes, updated.